components/nexus-core/src/main/java/org/sonatype/nexus/internal/httpclient/DefaultsCustomizer.java

```diff
@@ -58,6 +61,9 @@ public DefaultsCustomizer(
     this.requestTimeout = checkNotNull(requestTimeout);
     log.debug("Request timeout: {}", requestTimeout);
 
+    this.connectionRequestTimeout = checkNotNull(connectionRequestTimeout);
+    log.debug("Connection request timeout: {}", connectionRequestTimeout);
```

Comment:
blah, we have this pattern all over for component initialization, I don't see any value changing this.